tests/k8s/conftest.py

```diff
@@ -19,10 +21,44 @@ def __init__(self, kind_cluster: KindCluster, logger: JinaLogger):
         )
         self._log = logger
         self._set_kube_config()
+        self._install_linkderd(kind_cluster)
         self._loaded_images = set()
 
+    def _install_linkderd(self, kind_cluster):
+        self._log.info('Installing Linkerd to Cluster...')
+        proc = subprocess.Popen(
+            [f'{Path.home()}/.linkerd2/bin/linkerd', 'install'],
+            stdout=subprocess.PIPE,
+            env={"KUBECONFIG": str(kind_cluster.kubeconfig_path)},
+        )
+        kube_out = subprocess.check_output(
+            (
+                str(kind_cluster.kubectl_path),
+                'apply',
+                '-f',
+                '-',
+            ),
+            stdin=proc.stdout,
+            env=os.environ,
+        )
+        self._log.info('Poll status of linkerd install')
+        returncode = proc.poll()
+        self._log.info(
+            f'Installing Linkerd to Cluster returned code {returncode}, kubectl output was {kube_out}'
+        )
+        if returncode is not None and returncode != 0:
+            raise Exception(f"Installing linkerd failed with {returncode}")
+
+        self._log.info('check linkerd status')
+        out = subprocess.check_output(
+            [f'{Path.home()}/.linkerd2/bin/linkerd', 'check'],
+            env=os.environ,
+        )
+
+        print(f'linkerd check yields {out.decode() if out else "nothing"}')
```

Comment:
fyi: thats intentional, because the output hier is some specially formatted, multi line output. Our logger does not print it correctly so I use print here. Its just to understand what the test is doing so I think its not a big deal